.travis.yml

```diff
@@ -25,6 +25,23 @@ env:
 matrix:
   fast_finish: true
   include:
+    - name: icc
+      compiler: icc
+      env:
+      - C_COMPILER=icc
+      - CXX_COMPILER=icpc
+      - COMPILER_FLAGS='-wd13200 -wd13203'
```

Comment:
Just for my information,
what is this compiler flag about ?

Reply:
Ah I'm pretty sure they are to disable warnings when using MMX instructions improperly or something like that. Saw that in some other travis.yml files where they were using icc and blindly copied:/

Actually looking into it, I'm pretty sure we don't need it in zstd since we don't use the relevant calls. I'll go head and remove it.